<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>351</width>
                <height>230</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Login do Controle do Acervo</string>
        </property>
        <property name="sizeGripEnabled">
            <bool>false</bool>
        </property>
        <property name="modal">
            <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="QFrame" name="login_frame">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Plain</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QFormLayout" name="formLayout">
                                <item row="0" column="0">
                                    <widget class="QLabel" name="label">
                                        <property name="text">
                                            <string>Servidor :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="0" column="1">
                                    <widget class="QLineEdit" name="server">
                                        <property name="minimumSize">
                                            <size>
                                                <width>216</width>
                                                <height>32</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="0">
                                    <widget class="QLabel" name="label_2">
                                        <property name="text">
                                            <string>Usu√°rio :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="1" column="1">
                                    <widget class="QLineEdit" name="user">
                                        <property name="minimumSize">
                                            <size>
                                                <width>216</width>
                                                <height>32</height>
                                            </size>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="0">
                                    <widget class="QLabel" name="label_3">
                                        <property name="text">
                                            <string>Senha :</string>
                                        </property>
                                    </widget>
                                </item>
                                <item row="2" column="1">
                                    <widget class="QLineEdit" name="password">
                                        <property name="minimumSize">
                                            <size>
                                                <width>216</width>
                                                <height>32</height>
                                            </size>
                                        </property>
                                        <property name="echoMode">
                                            <enum>QLineEdit::Password</enum>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="QCheckBox" name="remember_me">
                                <property name="text">
                                    <string>Lembrar das credenciais</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QLabel" name="version_text">
                    <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QPushButton" name="cancelBtn">
                            <property name="text">
                                <string>&amp;Cancelar</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="submitBtn">
                            <property name="text">
                                <string>&amp;Confirmar</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>server</tabstop>
        <tabstop>user</tabstop>
        <tabstop>password</tabstop>
        <tabstop>remember_me</tabstop>
        <tabstop>submitBtn</tabstop>
        <tabstop>cancelBtn</tabstop>
    </tabstops>
    <resources />
    <connections />
</ui>